autoblocks/_impl/prompts/autogenerate.py

```diff
@@ -328,7 +353,18 @@ def generate_code_for_config(config: AutogeneratePromptsConfig) -> str:
     resp.raise_for_status()
     data = resp.json()
 
-    prompts = make_prompts_from_api_response(data, config)
+    # Get undeployed prompts as well
+    for prompt_id in set(p.id for p in config.prompts):
+        resp = httpx.get(
+            f"{API_ENDPOINT}/prompts/{prompt_id}/major/{UNDEPLOYED}/minor/{UNDEPLOYED}",
+            headers={"Authorization": f"Bearer {AutoblocksEnvVar.API_KEY.get()}"},
+        )
```

Comment:
Conceptually - what does `deploying` a prompt mean? Looking at prompt management in the UI - are we adding new options there?

Struggling with the use case of why I wouldn't just use a major version of 0

Reply:
let's do a call to discuss